js/src/interface.js

```diff
@@ -274,6 +266,30 @@ let app = new Vue({
     }
   },
   methods: {
+    initDataSourceProvider(currentDataSourceId) {
+      const $vm = this;
+      let dataSourceData = {
+        dataSourceTitle: 'Select the data source containing the user information',
```

Comment:
The data source title and default configurations depend on what's initialising the data source query provider, e.g. charts would call it with a different title. It should therefore not be hardcoded.